integration-tests/src/test/resources/bash-scripts/setup_wdt.sh

```diff
@@ -0,0 +1,258 @@
+#!/bin/bash
+# Copyright (c) 2018, 2021, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# Description:
+#
+#   This script generates a domain by calling the WebLogic Deploy Tool (WDT)
+#   bin/createDomain.sh script.
+#
+#   It first installs WDT using the given download URLs, and then runs it using the
+#   given Oracle home, WDT model file, WDT vars file, etc.
+#
+# Usage:
+#
+#   Export customized values for the input shell environment variables as needed
+#   before calling this script.   
+#
+# Outputs:
+#
+#   WDT install:           WDT_DIR/weblogic-deploy/...
+#
+#   Copy of wdt model:     WDT_DIR/$(basename WDT_MODEL_FILE)
+#   Copy of wdt vars:      WDT_DIR/$(basename WDT_VAR_FILE)
+#
+#   WDT logs:              WDT_DIR/weblogic-deploy/logs/...
+#   WDT stdout:            WDT_DIR/createDomain.sh.out
+#
+#   WebLogic domain home:  DOMAIN_HOME_DIR
+#                          default: /shared/domains/<domainUID>
+#
+# Input environment variables:
+#
+#   ORACLE_HOME    Oracle home with a WebLogic install.
+#                  default:  /u01/oracle
+#
+#   WDT_MODEL_FILE Full path to WDT model file.
+#                  default:  the directory that contains this script
+#                            plus "/wdt_model.yaml"
+#
+#   WDT_VAR_FILE   Full path to WDT variable file (java properties format).
+#                  default:  the directory that contains this script
+#                            plus "/create-domain-inputs.yaml"
+#
+#   WDT_INSTALL_ZIP_FILE  Filename of WDT install zip.
+#                  default:  weblogic-deploy.zip
+#
+#   WDT_INSTALL_ZIP_URL   URL for downloading WDT install zip
+#                  default:  https://github.com/oracle/weblogic-deploy-tooling/releases/download/release-1.9.7/$WDT_INSTALL_ZIP_FILE
+#
+#   https_proxy    Proxy for downloading WDT_INSTALL_ZIP_URL.
+#                  default: "http://www-proxy-hqdc.us.oracle.com:80"
```

Comment:
Remove the reference to http://www-proxy-hqdc.us.oracle.com:80

Reply:
removed